debug_gym/llms/human.py

```
@@ -470,6 +471,12 @@ def __init__(
         if prompt_toolkit_available:
             self._history = InMemoryHistory()
 
+        # Warn if self.logger.level is not set at least to INFO, as this is a human interface.
+        if self.logger.level > logging.INFO:
```

Is this required for the human mode to work / users to see the outputs? If so, should we raise an exception instead of just logging a warning?